griptape/drivers/web_scraper/base_web_scraper_driver.py

```diff
@@ -0,0 +1,8 @@
+from abc import ABC, abstractmethod
+from typing import Optional
+
+
+class BaseWebScraperDriver(ABC):
+    @abstractmethod
+    def scrape_url(self, url: str, *args, **kwargs) -> str:
```

Comment:
Do we need `*args` and `**kwargs`?

Reply:
I believe that @vasinov chose this interface to keep the options open in the derived classes, though I think we should be able to pass them via the constructor args instead.

Actually this is what we did in the derived classes, I'll either need to remove the `include_links` parameter from `Webloader.extract_page`, or start using it in the derived classes and remove the constructor args.

I would prefer moving `include_links` to a constructor arg because it is not determined via a parameter in the `WebScraper` tool, so I see a reason to pass it to `Webloader.extract_page` on each invocation.

Reply:
Removed `*args` and `**kwargs` from method signature